JASP-Engine/JASP/R/ttestindependentsamples.R

```diff
@@ -241,8 +241,13 @@ TTestIndependentSamples <- function(jaspResults, dataset = NULL, options, ...) {
         
         if (!isTryError(result))
           row <- c(row, result[["row"]])
-        else
+        else {
           errorMessage <- .extractErrorMessage(result)
+
+          if (result[["leveneViolated"]])
```

Comment:
@vandenman: Shouldn't this go in the `if (!isTryError(result))` block?
Otherwise `result[["LeveneViolated"]]` won't exist anyway

Reply:
you're right, it should actually be in the try.